esp-hal/src/trace.rs

```diff
@@ -203,14 +207,30 @@ where
     }
 }
 
+impl<T> Drop for Trace<'_, T>
+where
+    T: Instance,
+{
+    fn drop(&mut self) {
+        PeripheralClockControl::enable(self.peripheral.peripheral(), false);
```

Comment:
Might be just me but `set_enable(false)` or `disable()` would be clearer that `enable(false)` .

Reply:
I have no preference but I'm happy to change it if we agree on something else